contrib/requirements/requirements-hw.txt

```
@@ -6,7 +6,8 @@ hidapi<0.11
 trezor[hidapi]>=0.13.0,<0.14
 safet>=0.1.5
 keepkey>=6.3.1
-btchip-python>=0.1.32
+btchip-python>=0.1.32 # used for legacy Ledger Bitcoin app up to versions 2.0.* or below
+ledger-bitcoin=0.1.1 # used for new Ledger Bitcoin app >= v2.1.0 and above
```

CI is failing because this is not valid pip/setuptools syntax, you need `==`
```suggestion
btchip-python>=0.1.32  # used for legacy Ledger Bitcoin app up to versions 2.0.* or below
ledger-bitcoin==0.1.1  # used for new Ledger Bitcoin app >= v2.1.0 and above
```
but I would like to have a range specified here, e.g. 
`ledger-bitcoin>=0.1.1` or `ledger-bitcoin>=0.1.1,<0.2`